cmd/tsbs_generate_queries/databases/mongo/finance.go

```diff
@@ -0,0 +1,437 @@
+package mongo
+
+import (
+	"encoding/gob"
+	"fmt"
+	"time"
+
+	"github.com/timescale/tsbs/cmd/tsbs_generate_queries/uses/finance"
+	"github.com/timescale/tsbs/pkg/query"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func init() {
+	gob.Register(bson.A{})
+}
+
+type Finance struct {
+	*BaseGenerator
+	*finance.Core
+}
+
+func (f *Finance) LastPrice(q query.Query) {
+	query := q.(*query.Mongo)
+	query.Pipeline = mongo.Pipeline{
+		{
+			{"$sort", bson.D{
```

Comment:
Can you remind me what's the reason for adding the $sort stage? I remember you mentioned that during standup at some point.

Reply:
Ok, sounds good.